tests/commands/test_cmd_cron.py

```diff
@@ -70,9 +70,9 @@ def test_set_multiple(mock_chronos_add, mock_chronos_list, runner):
     _tmpl_path_2 = 'tests/fixtures/templates/chronos/test-chronos-2.json.tmpl'
     result = runner(
         [
-         'cron', 'set',
-         '--template', _tmpl_path,
-         '--template', _tmpl_path_2,
+            'cron', 'set',
+            '--template', _tmpl_path,
+            '--template', _tmpl_path_2,
```

Comment:
I have no idea how this slipped through in a previous commit, why didn't the linter catch it? 😕 

Not related to the main subject of this PR at all, just fixed to make tests pass.